config.toml

```diff
@@ -1,3 +1,4 @@
+baseURL = "https://ipfs.io/blog"
```

Comment:
Looks that it works as expected. Could you qualify what "changed the URLs to be relative when they should be." to you to ensure we are on the same page?